clean.js

```diff
@@ -0,0 +1,30 @@
+var fs = require('fs');
+
+function deleteFolderRecursive(path) {
+  if (fs.existsSync(path) && fs.lstatSync(path).isDirectory()) {
+    fs.readdirSync(path).forEach(function(file, index){
+      var curPath = path + "/" + file;
+
+      if (fs.lstatSync(curPath).isDirectory()) { // recurse
+        deleteFolderRecursive(curPath);
+      } else { // delete file
+        fs.unlinkSync(curPath);
+      }
+    });
+
+    console.log(`Deleting directory "${path}"...`);
+    fs.rmdirSync(path);
+  }
+};
+
+console.log("Cleaning working tree...");
+
+deleteFolderRecursive(".shadow-cljs");
```

Comment:
Could we `rm -rf .shadow-cljs; rm -rf .node_modules ...` in bash? It seems like this could be a one-line script in the package.json

Reply:
Done 👍